imemo.c

```diff
@@ -316,6 +316,7 @@ rb_imemo_mark_and_move(VALUE obj, bool reference_updating)
         else {
             if (vm_cc_super_p(cc) || vm_cc_refinement_p(cc)) {
                 rb_gc_mark_movable((VALUE)cc->cme_);
+                rb_gc_mark_movable((VALUE)cc->klass);
```

Comment:
Is this fix related to the forwarding change?

Reply:
@jeremyevans no, I think it's a bug in upstream. We happened to hit it in work CI so I fixed it here.

Reply:
No problem including it in this PR, but if you could make it a separate commit, I think that would be best.

Reply:
@jeremyevans I've pulled it in to a separate commit! Thanks for catching this. 😄